docs/reference/config-params.md

```diff
@@ -280,462 +324,543 @@ Otherwise it will default to `1`.
+### x-k8s.workload.livenessProbe.period
 
-Defines how many how long to wait before the first liveness probe runs for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the wait time from the information specified in the compose file.
+Defines how often liveness probe should run for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the interval from the information specified in the compose file.
 
-Kev uses the following heuristics to derieve that information for each service:
+Kev uses the following heuristics to derive that information for each service:
 
-If compose file(s) specifies the `healthcheck.start_period` attribute key in a service config it will use its value.
-Otherwise it'll default to `1m` (1 minute).
+If compose file(s) specifies the `healthcheck.interval` attribute key in a service config it will use its value.
+Otherwise, it'll default to `1m` (1 minute).
 
-### Default: `1m`
+#### Default: `1m`
 
-### Possible options: Arbitrary time duration. Example: `1m30s`
+#### Possible options: Time duration
 
-> kev.workload.liveness-probe-initial-delay:
+> x-k8s.workload.livenessProbe.period:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-initial-delay: 10s
+    x-k8s:
+      workload:
+        livenessProbe:
+          ...
+          period: 1m0s
 ...
 ```
 
-## kev.workload.liveness-probe-timeout
+### x-k8s.workload.livenessProbe.timeout
 
 Defines the timeout for the liveness probe for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the timeout value from the information specified in the compose file.
 
-Kev uses the following heuristics to derieve that information for each service:
+Kev uses the following heuristics to derive that information for each service:
 
 If compose file(s) specifies the `healthcheck.timeout` attribute key in a service config it will use its value.
-Otherwise it'll default to `10s` (10 seconds).
+Otherwise, it'll default to `10s` (10 seconds).
 
-### Default: `10s`
+#### Default: `10s`
 
-### Possible options: Arbitrary time duration. Example: `30s`
+#### Possible options: Arbitrary time duration. Example: `30s`
 
-> kev.workload.liveness-probe-timeout:
+> x-k8s.workload.livenessProbe.timeout:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-timeout: 10s
+    x-k8s:
+      workload:
+        livenessProbe:
+          ...
+          timeout: 30s
 ...
 ```
 
-## kev.workload.readiness-probe-type
+## x-k8s.workload.readinessProbe
```

Comment:
`readinessProbe` settings will match those of livenessProbe so worth checking whether they are on par. Alternatively could rework that section and point at relevant `livenessProbe` config sections with a note that `readinessProbe` should be used as top level key?

Reply:
They're on par.